docs/csharp/language-reference/builtin-types/snippets/shared/builtin-types.csproj

```diff
@@ -9,4 +9,8 @@
     <Platforms>AnyCPU;x86</Platforms>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile></DocumentationFile>
+  </PropertyGroup>
```

Comment:
@BillWagner what is the purpose of this addition to the project file, or this is accidental?